Presentation/Helper/Navigation/NavigationCoordinator.swift

```diff
@@ -71,7 +71,7 @@ final class NavigationCoordinator: CoordinatorProtocol {
                     ),
                     pageListViewModel: .init(pageListUseCase: DefaultPageListUseCase()),
                     searchViewModel: .init(),
-                    indexViewModel: .init(indexUseCase: DefaultIndexUseCase())
+                    indexViewModel: .init(indexUseCase: DefaultIndexUseCase()), backPageBtnViewModel: .init(BackPageBtnUsecase: DefaultBackPageBtnUseCase())
```

Comment:
이런 방식의 코드 컨벤션이 가독성 향상에 도움이 됩니다!! 특히 작은 화면일 때 더욱!

```suggestion
                    indexViewModel: .init(indexUseCase: DefaultIndexUseCase()),
                    backPageBtnViewModel: .init(BackPageBtnUsecase: DefaultBackPageBtnUseCase())
```